engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/db/PeriodicProcessesRepository.scala

```diff
@@ -309,7 +310,7 @@ class SlickPeriodicProcessesRepository(
         (
           rowNumber() :: Over
             .partitionBy((deployment.periodicProcessId, deployment.scheduleName))
-            .sortBy(deployment.runAt.desc),
+            .sortBy(deployment.runAt.desc, deployment.createdAt.desc),
```

Comment:
should we use `DeploymentStatus.ordering` here too?

Reply:
It is slick query - the `sortBy` gets `slick.lifted.Ordered`

Reply:
oh, right. Didn't notice. Sorry. 
One more question: do we need to sort in the DB too? Do these two sorting methods (in the service and in the repository) are related?

Reply:
I think it is since it came on tests in another project that is was required as well - still I am not 100% sure how exactly. I thought though, that since we are sorting here by `runAt` we should add `createdAt` as well since someone might use E.g. `getLatestDeploymentsForActiveSchedules` directly and it would be inconsistent with other methods.

Reply:
so, maybe at least we should leave here a comment that it should be changed when `DeploymentStatus.ordering` is changed. It would be great to have a dynamic ordering here, but AFAIU this is not simple. The comment seems to be an acceptable compromise.

Reply:
Good idea, I've added appropriate comment :+1: